exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -18,111 +18,151 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
-	"fmt"
-	"sync"
-	"time"
+	"github.com/google/uuid"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cloudwatch"
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.uber.org/zap"
+	"sync"
+	"time"
 )
 
 type exporter struct {
-	config *Config
+	config config.Exporter
 	logger *zap.Logger
-
+	retryCnt      int
```

Comment:
Please consider renaming this variable, either abbreviate both words or neither.

Not sure if this is my Australian nature showing through, but I would opt for the latter.